inst/extdata/emissions/FUT_EMISS_GV.csv

```diff
@@ -0,0 +1,171 @@
+# File: FUT_EMISS_GV.csv
+# Title: Future atmospheric abundances and climate forcings from scenarios of global and regional hydrofluorocarbon
+# Units: "NA"
```

Comment:
gotcha! then the comment isn't totally correct. Abundances and climate forcings would imply some sort of units.